packages/input/src/Component.tsx

```diff
@@ -35,6 +35,11 @@ export type InputProps = Omit<InputHTMLAttributes<HTMLInputElement>, 'size' | 't
      */
     type?: 'number' | 'card' | 'email' | 'money' | 'password' | 'tel' | 'text';
 
+    /**
+     * Ref для обертки input
+     */
+    wrapperRef?: React.MutableRefObject<HTMLDivElement> | null;
```

Comment:
Разве не так должно быть?

```suggestion
    wrapperRef?: React.MutableRefObject<HTMLDivElement | null>;
```

Reply:
да, спасибо)